includes/Classifai/Providers/Watson/NLU.php

```diff
@@ -1048,6 +1049,42 @@ public function add_process_content_meta_to_rest_api() {
 		);
 	}
 
+	/**
+	 * Display a dismissable admin notice when a threshold still at the default 70% and IBM Watson NLU active
+	 */
+	public function thresholds_update_notice() {
+		$config = $this->is_configured();
+
+		$thresholds = [
+			$config['features']['category_threshold'] ?? 55,
+			$config['features']['keyword_threshold'] ?? 55,
+			$config['features']['entity_threshold'] ?? 55,
+			$config['features']['concept_threshold'] ?? 55,
+		];
+
+		if ( ! in_array( 70, $thresholds, true ) ) {
+			return;
+		}
```

Comment:
This doesn't work because `is_configured` will return a `boolean`, indicating if the feature is enabled or not. But we take that value and try and get the threshold settings from it, which won't work. We need to check if the feature is configured and then get the settings for the feature to look at